### PR TITLE
iyjian-我用wechaty做了一个积分红包机器人

### PR DESCRIPTION
**Pull request by iyjian**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.
```
---
 layout: post
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2017` directory.
- [x] Add `<!-- more -->` section to just show the abstract in blog homepage.

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

File: _posts/2017-11-08-red-pocket-wechaty-iyjian.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+最后，谢谢你用这么多时间看了我写的字。
+
+
+
+
+
+  [1]: /download/2017/iyjian-1.png
+  [2]: /download/2017/iyjian-2.png
+  [3]: /download/2017/iyjian-5.jpg
```
Comment (lijiarui) [inline _posts/2017-11-08-red-pocket-wechaty-iyjian.md]: I found you just have 3 images, could you change `iyjian-5.jpg` to `iyjian-3.jpg` for a better looking?
Thanks.

File: _posts/2017-11-08-red-pocket-wechaty-iyjian.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## 一些杂七杂八的事情
+
+微信桌面客户端（我用的是mac)和微信网页版对视频poster的提取是不一样的，我的合伙人是个做产品的，他做完视频以后会让poster上显示一个他觉得满意的画面，每次都是mac客户端发给我，看起来都是赞赞的，可是我放到wechaty里发出去以后并不是显示的那一帧poster，找了很久原因才想到两个版本的poster提取方法不一样。
+docker容器中默认的时间不是东八区，一定要重新搞一下，我吃了很多亏，数据库和服务器上报的时间不一致，但是我刚写这个东西的时候发现已经支持时区更改了：[Add timezone in to Dockerfile](https://github.com/Chatie/wechaty/issues/594)。
+我开始写代码的时候把业务逻辑和wechaty对微信自动化的逻辑都写到一个项目了，这样维护的时候很痛苦，改动一个业务逻辑就必须重启一次wechaty，会有服务中断，后来把业务逻辑包装成http请求，wechaty里只做简单的收发消息，踢人，拉人，自动好友等操作，把消息发给业务逻辑，然后再得到指令后操作，这样可以维护的方便一点。
+群邀请也是需要养的，新建的一个群别指望发群邀请把人拉进来，最开始无法拉人，需要给用户回复一张群二维码让他扫码进群，等人多一点，但是不要等到100个人，因为100个人以后就只能通过邀请才能加人了。
+
+先写这么多吧，最后说一下，我的个人号已经于9月份全军覆没了，正好是我过生日的那天，还是在三亚度假的时候，我整个人都有点懵了，但仔细想想一点都不可惜，有了这些经验东山再起不是什么问题，我和那个产品经理已经重新开始弄公众号了，圈子里有了品牌，用户做起来是分分钟的事情，待把公众号做稳一点了，还是会继续实施上面的公众号和个人号互动的想法，应该没那么容易被封了，不会傻乎乎的一个号上整那么多人，给自己找那么多麻烦了。
+以上这么多废话是答应lijiarui小姐9月份写的，可是又要忙工作，又要忙这个娱乐项目，每天都是凌晨2，3点睡觉，6，7点起床，直到现在也如此，实在没有什么完整的时间，对于这种严重的拖延我深表歉意。
```
Comment (lijiarui) [inline _posts/2017-11-08-red-pocket-wechaty-iyjian.md]: Could you help to add my GitHub link here? [lijiarui](github.com/lijiarui) Thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks for your blog! Could you help to fix these small change, then I will merge this.

**Comment by iyjian** [comment]
done

**Pull request merged**